app/src/main/java/mozilla/lockbox/presenter/FingerprintDialogPresenter.kt

```diff
@@ -42,7 +43,10 @@ class FingerprintDialogPresenter(
             .addTo(compositeDisposable)
 
         view.cancelTapped
-            .subscribe { view.onCancel() }
+            .subscribe {
+                dispatcher.dispatch(FingerprintAuthAction.OnCancel)
+                view.onCancel()
```

Comment:
nit: I know this method wasn't implemented as part of this PR, but I just noticed the name & I feel like `dismiss` is more descriptive

Reply:
I agree :) I will change it